travis-ci/init-integration-tests.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
+
+set -ex
```

Comment:
The `-x` is going to make the output a lot more verbose.  That's fine if it's being done intentionally, just want to make sure that this wasn't a debug flag that was left in.

Reply:
Left in from the original integration test file from when i split it up. Do you think we need the `-ex` somewhere?

Reply:
`set -e` is sufficient.